Support for Monstargear XO87 Hot-Swap PCB (#11555)
qmk/qmk_firmware: keyboards/xo87/rgb/readme.md

# XO87 RGB PCB

![XO87](https://cdn.imweb.me/thumbnail/20201120/c90a5bdb75264.png) 

> A TKL custom keyboard made by Monstargear 

Reviewer: ```suggestion
A TKL custom keyboard made by Monstar Gear.
```

> * Keyboard Maintainer: qmk

Reviewer: ```suggestion
* Keyboard Maintainer: The QMK Community
```

* Hardware Supported: XO87 RGB Hot-Swap version
* Hardware Availability: [Monstargear](https://monstargears.com)

Make example for this keyboard (after setting up your build environment):

> ```sh
> make xo87/rgb:default        # builds the default firmware
> make xo87/rgb:via            # builds the via firmware
> ```
> The keyboard uses a DFU bootloader.<br>
> To enter DFU mode, hold the escape key while plugging in your keyboard.<br>
> To make a keymap and use dfu to flash it:
> 
> ```sh
> make xo87/rgb:default:flash  # builds and flashes the default version
> make xo87/rgb:via:flash      # builds and flashes the via version
> ```

Reviewer 2: ```suggestion
    make xo87/rgb:default

Flashing example for this keyboard:

    make xo87/rgb:default:flash
```

> Please be aware that Monstargear has not yet switched to this firmware and any PCB that is running the pre-installed firmware will need to use ISP to flash their board for the first time.

Reviewer: ```suggestion
Please be aware that Monstar Gear has not yet switched to this firmware and any PCB that is running the pre-installed firmware will need to use ISP to flash their board for the first time.
```
